src/components/EntityInfo/EntityMenu.vue

```diff
@@ -110,7 +110,7 @@
             <v-tooltip right content-class="right-tooltip">
               <template v-slot:activator="{ on }">
                 <v-list-item
-                  v-if="isBenBcCccUlc"
+                  v-if="isBenBcCccUlc || isCoop"
```

Comment:
This shows the list item. I think you had it right before :)

Reply:
Did we say this will never apply to Coops, or we want to follow the same logic as the staff menu item?

Reply:
Same logic for now. Based on that it is disabled for COOPs